Vic2ToHoI4/Source/V2World/Military/Leaders/TraitGroup.cpp

```diff
@@ -0,0 +1,14 @@
+#include "TraitGroup.h"
+#include "ParserHelpers.h"
+#include "Trait.h"
+
+
+
+Vic2::TraitGroup::TraitGroup(std::istream& theStream)
+{
+	registerRegex(commonItems::catchallRegex, [this](const std::string& traitName, std::istream& theStream) {
```

Comment:
This here throws warnings about shadowing theStream name. It's why I prefer to use registerKeys() function to separate init from the keys themselves.